specification/search/data-plane/Azure.Search/stable/2024-07-01/searchservice.json

```diff
@@ -0,0 +1,11543 @@
+            "name": "TextEmbedding3Small"
+          }
+        ]
+      },
+      "description": "The Azure Open AI model name that will be called."
+    },
+    "CustomVectorizer": {
+      "type": "object",
+      "x-ms-discriminator-value": "customWebApi",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VectorSearchVectorizer"
+        }
+      ],
+      "properties": {
+        "customWebApiParameters": {
+          "x-ms-client-name": "CustomWebApiParameters",
+          "$ref": "#/definitions/CustomWebApiParameters",
+          "description": "Specifies the properties of the user-defined vectorizer."
+        }
+      },
+      "description": "Specifies a user-defined vectorizer for generating the vector embedding of a query string. Integration of an external vectorizer is achieved using the custom Web API interface of a skillset."
+    },
+    "CustomWebApiParameters": {
+      "type": "object",
+      "properties": {
+        "uri": {
+          "type": "string",
+          "format": "uri",
+          "description": "The URI of the Web API providing the vectorizer."
+        },
+        "httpHeaders": {
+          "$ref": "#/definitions/WebApiHttpHeaders",
+          "description": "The headers required to make the HTTP request."
+        },
+        "httpMethod": {
+          "type": "string",
+          "description": "The method for the HTTP request."
+        },
+        "timeout": {
+          "type": "string",
+          "format": "duration",
+          "description": "The desired timeout for the request. Default is 30 seconds."
+        },
+        "authResourceId": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "Applies to custom endpoints that connect to external code in an Azure function or some other application that provides the transformations. This value should be the application ID created for the function or app when it was registered with Azure Active Directory. When specified, the vectorization connects to the function or app using a managed ID (either system or user-assigned) of the search service and the access token of the function or app, using this value as the resource id for creating the scope of the access token."
+        },
+        "authIdentity": {
+          "$ref": "#/definitions/SearchIndexerDataIdentity",
+          "x-nullable": true,
+          "description": "The user-assigned managed identity used for outbound connections. If an authResourceId is provided and it's not specified, the system-assigned managed identity is used. On updates to the indexer, if the identity is unspecified, the value remains unchanged. If set to \"none\", the value of this property is cleared."
+        }
+      },
+      "description": "Specifies the properties for connecting to a user-defined vectorizer."
+    },
+    "VectorSearchVectorizerKind": {
+      "type": "string",
+      "enum": [
+        "azureOpenAI",
+        "customWebApi"
```

Comment:
Is this missing AML and AI Services Vision as vectorizer choices?

Reply:
No, we are not GAing those options in this release, they are staying preview for now.